contracts/test/hardhat/contracts/FleekERC721/token-roles.t.ts

```diff
@@ -124,86 +104,110 @@ describe('FleekERC721.TokenRoles', () => {
     await contract.transferFrom(owner.address, otherAccount.address, tokenId);
 
     expect(await contract.ownerOf(tokenId)).to.equal(otherAccount.address);
-    expect(
-      await contract.hasTokenRole(tokenId, Roles.Owner, otherAccount.address)
-    ).to.be.true;
-    expect(await contract.hasTokenRole(tokenId, Roles.Owner, owner.address)).to
-      .be.false;
   });
 
   it('should clean the token controller list after transfer', async () => {
     const { contract, owner, otherAccount, tokenId } = fixture;
     await contract.grantTokenRole(
       tokenId,
-      Roles.Controller,
+      TokenRoles.Controller,
       otherAccount.address
     );
     await contract.transferFrom(owner.address, otherAccount.address, tokenId);
 
-    expect(await contract.getTokenRoleMembers(tokenId, 1)).to.eql([]);
+    expect(
```

Comment:
This comment is not needed to be done in this PR but just noting, thinking we should do a test where we add more than one controllers and then check they are all not on the list.